lib/pflua/src/pf/utils.lua

```diff
@@ -11,6 +11,11 @@ struct pflua_timeval {
 int gettimeofday(struct pflua_timeval *tv, struct timezone *tz);
 ]]
 
+-- Additional function arguments other than 'len' and 'P' that the
+-- compiled function may accept (this needs to be tracked in several
+-- pflua passes, which is why the data is kept here)
+extra_args = {}
```

Comment:
I certainly would rather not pollute the global namespace. Given that the value is a table, the variable can be localized *as long as it is never reasiigned* (the table items can be cleared, though). That would be okay to avoid always referencing it as `utils.extra_args`, adding a comment next to the definition explaining that it shouldn't be reassigned.